requirements.txt

```diff
@@ -1,6 +1,59 @@
-openai==1.55.3
+aiofiles==24.1.0
+aiohappyeyeballs==2.4.4
+aiohttp==3.11.10
+aiosignal==1.3.2
+annotated-types==0.7.0
+anyio==4.7.0
+attrs==24.3.0
+certifi==2024.12.14
+cffi==1.17.1
+charset-normalizer==3.4.0
+cryptography==44.0.0
```

Comment:
⚠️ Regularly check for updates to 'cryptography==44.0.0' due to potential security concerns.